examples/notebooks/Bezout's_Dixon's_resultants.ipynb

```diff
@@ -0,0 +1,970 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import sympy as sym\n",
+    "from sympy.polys import subresultants_qq_zz\n",
+    "\n",
+    "sym.init_printing()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Bezout matrix is a special square matrix associated with two polynomials, introduced by Sylvester (1853) and Cayley (1857) and named after Étienne Bézout. Bézoutian may also refer to the determinant of this matrix, which is equal to the resultant of the two polynomials.\n",
+    "\n",
+    "The entries of Bezout matrix are bilinear functions of coefficients of the given polynomials. The Bezout formulation has gone over different generalizations. The most common one is the Cayley that is used after for the Dixon  formulation as well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\left|\\begin{matrix}\n",
+    "p(x) & q(x)\\cr\n",
```

Comment:
This is showing "unknown character" in [GitHub](https://github.com/Nikoleta-v3/sympy/blob/b4ca77bed357e3d3091e9b70fc249b0a05895b6c/examples/notebooks/Bezout's_Dixon's_resultants.ipynb)